cmake/libint2-config.cmake.in

```diff
@@ -0,0 +1,245 @@
+# libint2-config.cmake
+# -------------------
+#
+# Libint2 cmake module.
+# This module sets the following variables in your project:
+#
+# ::
+#
+#   Libint2_FOUND - true if Libint2 and all required components found on the system
+#   Libint2_VERSION - Libint2 version in format Major.Minor.Release
+#   Libint2_INCLUDE_DIRS - Directories where Libint2 and libderiv headers are located.
+#   Libint2_INCLUDE_DIR - same as DIRS
+#   Libint2_LIBRARIES - Libint2 and libderiv libraries to link against.
+#   Libint2_LIBRARY - same as LIBRARIES
+#   Libint2_MAX_AM_ERI - maximum angular momentum level of Libint2 libraries
+#
+#
+# Available components:
+#
+# ::
+#
+#   shared - search for only shared library
+#   static - search for only static library
+#
+#   onebody_dD_lL    - search for library including 1-body integrals with derivative order D (D=0..4) and max angular momentum up to L (L=2..10)
+#   twobody_cC_dD_lL - search for library including 2-body integrals with C centers, derivative order D (D=0..4), and max angular momentum up to L (L=2..10)
+#
+#                   cart       shell_set  used_by
+#                   --------   ---------  -------
+#   ss - search for standard + standard = mpqc4, psi4
+#   so - search for          + orca
+#   is - search for intv3    + standard = mpqc3
+#   io - search for          + orca
+#   gs - search for gamess   + standard = gamess
+#   go - search for          + orca
+#   os - search for orca     + standard
+#   oo - search for          + orca     = orca
+#   bs - search for bagel    + standard = bagel
+#   bo - search for          + orca
+#
+#
+# Exported targets:
+#
+# ::
+#
+# If Libint2 is found, this module defines the following :prop_tgt:`IMPORTED`
+# targets. ::
+#
+##   Libint2::int2 - library only
+#   Libint2::cxx - library + C++11 API
+#
+#
+# Suggested usage:
+#
+# ::
+#
+#   find_package(Libint2)
+#   find_package(Libint2 2.7.0 CONFIG REQUIRED COMPONENTS shared ss twobody_c4_d0_l5 twobody_c4_d1_l4)
+#
+#
+# The following variables can be set to guide the search for this package:
+#
+# ::
+#
+#   Libint2_DIR - CMake variable, set to directory containing this Config file
+#   CMAKE_PREFIX_PATH - CMake variable, set to root directory of this package
+#   PATH - environment variable, set to bin directory of this package
+#   CMAKE_DISABLE_FIND_PACKAGE_Libint2 - CMake variable, disables
+#     find_package(Libint2) when not REQUIRED, perhaps to force internal build
+
+@PACKAGE_INIT@
+
+set(PN Libint2)  # ProjectName
+set(pnv libint2)  # projectnameversion
```

Comment:
Speaking for Psi4, I'm willing to change up all the p4/l2 cmake interactions if that could allow a single `libint2` or `Libint2` spelling rather than both. Appeasing cmake for simplification is worth it in my opinion.

Originally, I had chosen the unused `Libint2` as I thought there were enough options name changes, detection components additions, etc. to want to force downstream projects to make a clean break with the libtool generation or the previous export tarball. But if you'd prefer to opt for continuity, should we settle on `libint2`?